test/Microsoft.AspNetCore.Razor.Test.Common/Langauge/IntegrationTests/IntegrationTestBase.cs

```diff
@@ -184,9 +184,6 @@ protected void AssertDesignTimeDocumentMatchBaseline(RazorCodeDocument document)
             Assert.NotNull(syntaxTree);
             Assert.True(syntaxTree.Options.DesignTimeMode);
```

Comment:
Not really. Saw these guys too. Just figured I'd make the smallest changeset possible; can remove.